shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -472,6 +466,7 @@
 bool FlutterPlatformViewsController::SubmitFrame(GrContext* gr_context,
                                                  std::shared_ptr<IOSContext> ios_context,
                                                  std::unique_ptr<SurfaceFrame> frame) {
+  FML_DCHECK(flutter_view_);
```

Comment:
I think this one is good enough, I'm ok with removing the other ones.

Reply:
My thought is that these are free outside debug mode, and this will help clarify things if we refactor any of this in the future. WDYT?

Reply:
Sure, SG! We probably don't need the one in `PrerollCompositeEmbeddedView`, as it seems this method doesn't use `flutter_view_` at all.

Reply:
Done